hr_employee_cost_history/views/hr_employee_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L. <info@moduon.team>
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+    <record
+        id="hr_timesheet_retroactive_cost_view_employee_form_inherit"
+        model="ir.ui.view"
+    >
+        <field name="name">hr.employee.view.form.inherit</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form" />
+        <field name="arch" type="xml">
+
+            <xpath expr="//field[@name='timesheet_cost']" position="attributes">
+                <attribute name="readonly">1</attribute>
+            </xpath>
+
+            <xpath expr="//field[@name='currency_id']" position="before">
+                <field name="timesheet_cost_history_ids" invisible="1" />
+                <button
+                    type="action"
+                    name="%(action_hr_employee_timesheet_cost_wizard)d"
+                    class="btn btn-link oe_inline"
+                    role="button"
+                    context="{'default_employee_id': id, 'default_timesheet_cost': timesheet_cost}"
+                >
+                    <i class="fa fa-fw o_button_icon fa-arrow-right" /> Update Cost
+                </button>
+                <button
+                    type="action"
+                    name="%(hr_employee_timesheet_cost_action)d"
+                    class="btn btn-link oe_inline"
+                    role="button"
+                    context="{'default_employee_id': id}"
+                    groups="hr_timesheet.group_timesheet_manager"
```

Comment:
issue: since I'm suggesting to keep the same permissions level as upstream, this should change too:

```suggestion
                    groups="hr.group_hr_user"
```